hack/tools/image/variants.yaml

```diff
@@ -0,0 +1,5 @@
+variants:
+  "1.20":
+    image: "golang:1.20.8"
+  "1.21":
+    image: "golang:1.21.1"
```

Comment:
How do we maintain them? Will dependabot take care?

Reply:
Unfortunately, Dependabot does not support updating the same image with different tags in one file.
Thus, we have to maintain them manually for now and look for a different solution.